server/views/pages/goal/complete/index.njk

```diff
@@ -30,10 +30,17 @@
                 Add a note (optional)
               </label>
             </h1>
-            <p class="govuk-hint">
-              Learning and work progress plans, including notes, are shared with prisoners and other staff
-            </p>
-            <textarea class="govuk-textarea" id="completed-note" name="note" rows="5" aria-describedby="warning-text">{{ form.note }}</textarea>
+            {{ govukTextarea({
```

Comment:
👍 Good shout for using the GDS component rather than writing the markup yourself 👍 
but I think the page will end up with 2 "Add a note (optional)" headings (1 from the H1 above, and 1 from the label attribute of this component)

Given that your page already (correctly) has a h1 ("Are you sure you want to mark this goal as completed?"), and we dont want a page with 2 H1's anyway, I'd be tempted to remove the h1 above, and keep the heading in the component (which is being marked up as a h2 👍 )

Reply:
oops - removed the additional h1